src/openrct2/network/NetworkBase.cpp

```diff
@@ -43,7 +43,7 @@
 // It is used for making sure only compatible builds get connected, even within
 // single OpenRCT2 version.
 
-#define NETWORK_STREAM_VERSION "18"
+#define NETWORK_STREAM_VERSION "19"
```

Comment:
This will need to be bumped again
```suggestion
#define NETWORK_STREAM_VERSION "20"
```

https://github.com/OpenRCT2/OpenRCT2/pull/20123 already claims v19

Reply:
Will rebase

Reply:
Actually that isn't merged yet, will rebase once the PR is merged.

Reply:
done